src/components/UserContext.tsx

```diff
@@ -1,4 +1,8 @@
-import React, { useContext } from 'react'
+import React, { useContext, useReducer, useEffect } from 'react'
+import useSWR, { mutate } from 'swr'
+import { fetchFromFirebase } from 'utils/fetcher'
+import firebase from 'firebase'
```

Comment:
Why do your import firebase from 'firebase'? 
Could it be 'lib/firebase'?

Reply:
Fixed